Few-iOS/Button.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Button.swift
+//  Few
+//
+//  Created by Coen Wessels on 13/03/15.
+//  Copyright (c) 2015 Josh Abernathy. All rights reserved.
+//
+
+import UIKit
+
+private class FewButton: UIButton {
```

Comment:
Yes it works the same, however the problem with `TargetActionTrampoline` is that the `addTarget` method of UIControl doesn't retain its target. So after you create a new `Button` the `TargetActionTrampoline` gets deallocated.